src/main.c

```diff
@@ -86,7 +86,10 @@ int main(int argc, char *argv[])
 
     /* check if dpvs is running and remove zombie pidfile */
     if (dpvs_running(DPVS_PIDFILE)) {
-        fprintf(stderr, "dpvs is already running\n");
+        fprintf(stderr, 
```

Comment:
If the binary is changed for example upgraded, the version shown by this line is not the `DPVS` version which is actually running. So user may get confused.

Reply:
Not really get it. Shouldn't the binary be re-compiled and restarted when upgrades. So the user will get the version actually running. Please correct me if I am wrong. Thanks.

Reply:
Assume the old version is 1.0, and it's already running. 
New image is 1.1, now try to execute 1.1, then it  will show
```
dpvs is already running
version 1.1, build on ....
```
User may think dpvs 1.1 is already running, but actually it's 1.0.

Reply:
yes, you're right. I've modified the PR by adding a application arguments parse before the EAL ones. Please review it, thanks!